alfresco-java-rest-api/alfresco-java-rest-api-lib/pom.xml

```diff
@@ -62,12 +62,6 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
-  <repositories>
```

Comment:
There was an additional repository definition in the rest-api library pom, we already define the repositories it the main [pom.xml](https://github.com/Alfresco/alfresco-java-sdk/blob/develop/pom.xml#L88-L113) so this is just noise and can be removed

Reply:
I think you need to put the alfresco-public one here instead in order to resolve the parent

Reply:
Fixed